pkg/lang/python/imports_test.go

```diff
@@ -19,44 +19,44 @@ func TestFindImports(t *testing.T) {
 			name:   "import module",
 			source: "import mymodule",
 			want: map[string]Import{
-				"mymodule": {Name: "mymodule", ImportedSelf: true},
+				"mymodule": {Name: "mymodule"},
 			},
 		},
 		{
 			name:   "import module aliased",
 			source: "import mymodule as m",
 			want: map[string]Import{
-				"mymodule": {Name: "mymodule", ImportedSelf: true, Alias: "m"},
+				"mymodule": {Name: "mymodule", Alias: "m"},
```

Comment:
I'm probably missing something, but `"."` seems odd as the key here. What does the key essentially represent? I _thought_ it was "the name that the code will refer to this as" — which here would be the alias, but that doesn't seem to be the case.

What happens if someone does:

```python
# klotho::whatever { id = "my_m" }
import mymodule as m
import mymodule as x

x.foo()
```

Wouldn't this key structure confuse `m` and `x`?

Reply:
FindImports is just the code parsing, so it's just `from <x> import <y>` would result in `Import{Name: <x>, Attributes: [<y>]}`.

The case you pointed out is tested in case `import multiple attributes from module in separate imports`

Reply:
Alright, works for me. I added the "bug" label to that ticket, because this really is a bug — not just a lacking feature. But we can discuss it in grooming.